### DISCUSSION TITLE
Not show 50Hz and 100Hz lowband on lua script.

### DISCUSSION THREAD

**Discussion by cronosrj** [ExpressLRS/ExpressLRS#2967] [General]
I'm using Radiomaster Nomad on my TX12 and when loading the lua script the 50Hz and100Hz lowband options are not displayed. I've already updated the module to version 3.5 as well as the lua script as well.

**Comment by pkendall64** [comment]
They are at the far end of the Lua script when scrolling the wheel pas 500 2.4, theres 50 Low Band, 100 Low Band 100 Full Low Band and 200 Low Band.

**Reply by cronosrj** [reply]
https://youtu.be/CigwtcbgqLM
As you can see it doesn't really display.

**Reply by pkendall64** [reply]
Can you show the version number at the bottom of the Lua script. I have just reflashed to the official 3.5.0 using the web-flasher
https://expresslrs.github.io/web-flasher/
https://www.youtube.com/shorts/Z_noFcUdJbA

**Reply by cronosrj** [reply]
Of course!

![IMG_20241004_014221541_HDR](https://github.com/user-attachments/assets/208a3a38-0f9a-465b-af65-34d06960dc72)

**Reply by pkendall64** [reply]
Erm, only other thing I could think of is the baud rate!? I have mine set to 5.25MBaud

**Reply by cronosrj** [reply]
> Erm, only other thing I could think of is the baud rate!? I have mine set to 5.25MBaud

Working perfectly!!! Thank you very much for the support.
